scripts/build_renewable_profiles.py

```diff
@@ -287,20 +288,22 @@
         if "natura" in config and config["natura"]:
             excluder.add_raster(paths.natura, nodata=0, allow_no_overlap=True)
 
-        copernicus = config.get("copernicus", {})
-        if "grid_codes" in copernicus:
-            codes = copernicus["grid_codes"]
-            excluder.add_raster(paths.copernicus,
-                                codes=codes,
-                                invert=True,
-                                crs=3035)
-        if copernicus.get("distance", 0.0) > 0.0:
-            codes = copernicus["distance_grid_codes"]
-            buffer = copernicus["distance"]
-            excluder.add_raster(paths.copernicus,
-                                codes=codes,
-                                buffer=buffer,
-                                crs=3035)
+        if "copernicus" in config and config["copernicus"]:
+            copernicus = config["copernicus"]
+            excluder.add_raster(
+                paths.copernicus,
+                codes=copernicus["grid_codes"],
+                invert=True,
+                crs="EPSG:4236",
+            )
+            if "distance" in copernicus and config["copernicus"][
+                    "distance"] > 0:
+                excluder.add_raster(
+                    paths.copernicus,
+                    codes=copernicus["distance_grid_codes"],
+                    buffer=copernicus["distance"],
+                    crs="EPSG:4236",
```

Comment:
Hey @euronion, I think we have a bug here. CRS should be EPSG:4326 ? Like typically provided: https://zenodo.org/record/3518038#.YoytIznMKEA